### PR TITLE
feat(plugin): Typescript plugin

### PR DESCRIPTION
**Pull request by RemyJeancolas**

### CODE REVIEW

**Review by wzrdtales — CHANGES_REQUESTED** [review]
UNIX File ending missing. please always end a file with an emtpy line.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      }
+    });
+
+    delete module.exports.loadPlugin;
+  },
+  name: 'typescript',
+  hooks: [
+    'migrator:migration:hook:require'
+  ]
+};
```
Comment (wzrdtales) [inline index.js]: UNIX File ending missing. please always end a file with an emtpy line.

File: .nmpignore [hunk 2]
```diff
@@ -0,0 +1 @@
+tests
```
Comment (wzrdtales) [inline .nmpignore]: UNIX File ending missing. please always end a file with an emtpy line.

**Review by wzrdtales — CHANGES_REQUESTED** [review]
UNIX File ending missing. please always end a file with an emtpy line.

**Review by wzrdtales — APPROVED** [review]

**Comment by RemyJeancolas** [comment]
My bad, my linter usually does it for me ^^
Fixed

**Comment by wzrdtales** [comment]
@RemyJeancolas No worries :)

**Comment by wzrdtales** [comment]
@RemyJeancolas As you're the author, do you want to be added to this repo?

**Comment by RemyJeancolas** [comment]
Yes with pleasure.

**Comment by wzrdtales** [comment]
https://www.npmjs.com/package/db-migrate-plugin-typescript

**Comment by wzrdtales** [comment]
added.

**Comment by RemyJeancolas** [comment]
Cool, if you want to add me as a collaborator on the npm repo, my nickname on npm is [ergy](https://www.npmjs.com/~ergy).

**Comment by wzrdtales** [comment]
Also added.

**Comment by RemyJeancolas** [comment]
Super efficient, thank you !

**Pull request merged**